src/api/entities/Asset/index.ts

```diff
@@ -437,9 +434,9 @@ export class Asset extends Entity<UniqueIdentifiers, string> {
   /**
    * Assign a new primary issuance agent for the Asset
    *
-   * @note this may create AuthorizationRequest which have to be accepted by
-   *   the corresponding Account. An Account or Identity can
-   *   fetch its pending Authorization Requests by calling `authorizations.getReceived`
+   * @note this will create an {@link AuthorizationRequest Authorization Request} which has to be accepted by the `target` Identity.
```

Comment:
```suggestion
   * @note this will create an {@link AuthorizationRequest | Authorization Request} which has to be accepted by the `target` Identity.
```